docs/notable-changes/1.22.1/zones-discovery.md

```diff
@@ -0,0 +1,80 @@
+# Updating Kyma Environment Broker – Zones Discovery
+
+> [!NOTE]
+> This is an optional change. Operators can enable Zones Discovery for AWS by updating the Kyma Environment Broker (KEB) configuration.
+
+## Prerequisites
+
+Ensure that:
+- The Kyma Environment Broker (KEB) is configured to use the AWS provider.
+
+## What's Changed
+
+The [Zones Discovery](../../contributor/03-55-zones-discovery.md) feature allows KEB to dynamically retrieve available zones from the hyperscaler (currently only AWS) instead of relying on statically configured zones in the `providersConfiguration`.
```

Comment:
```suggestion
With the [Zones Discovery](../../contributor/03-55-zones-discovery.md) feature, KEB can dynamically retrieve available zones from the hyperscaler (currently only AWS) instead of relying on statically configured zones in the `providersConfiguration`.
```